packages/mobile/src/screens/edit-track-screen/components/CollectibleGatedAvailability.tsx

```diff
@@ -71,16 +70,16 @@ const useStyles = makeStyles(({ typography, spacing, palette }) => ({
   },
   learnMore: {
     marginTop: spacing(4),
-    flexDirection: 'row',
-    alignItems: 'center'
+    ...flexRowCentered()
   },
   learnMoreText: {
     marginRight: spacing(1),
     fontFamily: typography.fontByWeight.bold,
     fontSize: typography.fontSize.large,
     color: palette.neutralLight4
   },
-  collectionContainer: {
+  selectionContainer: {
+    marginLeft: spacing(-10),
```

Comment:
Is this necessary? Will it display correctly on different screen sizes? Just checking cuz negative margins sets off alarms.

Reply:
yes this should be fine. it's because of the way the radio components work. the expansible context shouldn't be indented, unlike the title, hence this negative margin.